src/App.scss

```diff
@@ -1,8 +1,49 @@
+/* eslint-disable max-len */
+
 iframe {
   display: none;
 }
 
-html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
-  font-size: 24px;
+@mixin flex {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+body {
+  --bg-main: linear-gradient(90deg,
+  rgba(223,233,243,1) 0%, rgba(255,255,255,1) 100%);
+  --font-main: 24px;
+  padding-top: 20vh;
+
+  font-family: "Roboto", sans-serif;
+  font-size: var(--font-main);
+  letter-spacing: 3px;
+  color: #94b0da;
+  background: var(--bg-main);
+}
+
+.App {
```

Comment:
App is only one selector which is belong to the file `App.scss`,  other selectors should be in separated files like reset.scss, mixins.scss...

Reply:
fixed)